src/gui/guiFormSpecMenu.cpp

```diff
@@ -556,17 +560,30 @@ void GUIFormSpecMenu::parseScrollBar(parserData* data, const std::string &elemen
 
 		spec.ftype = f_ScrollBar;
 		spec.send  = true;
-		gui::IGUIScrollBar* e =
-				Environment->addScrollBar(is_horizontal,rect,this,spec.fid);
+		guiScrollBar *e = new guiScrollBar(Environment, this, spec.fid, rect,
```

Comment:
I see you didn't compile yet.
```suggestion
		GUIScrollBar *e = new GUIScrollBar(Environment, this, spec.fid, rect,
```